easyunfold/cli.py

```diff
@@ -159,10 +157,6 @@ def unfold(ctx, data_file, mpl_style_file):
     unfoldset = loadfn(data_file)
     click.echo(f'Loaded data from {data_file}')
     ctx.obj = {'obj': unfoldset, 'fname': data_file}
-    if mpl_style_file:
```

Comment:
So the reason I moved it from here was that it didn't show up in the `easyunfold unfold plot -h` message (where most people would be looking for something like this), and also had to be given in an odd format as 
```
easyunfold unfold --mpl-style-file X.mplstyle plot --ncl ...
```
(i.e. before the `plot` etc subcommands). 
I've updated this to make it a decorator now to avoid the code duplication but still show it as an option in the help messages for `effective_mass`, `plot` and `plot-projections` ✅